E2E-tests/.sops.yaml

```diff
@@ -0,0 +1,9 @@
+creation_rules:
+    - path_regex: secrets\/[^/]+\/[^l].*|secrets\/[^/]+\/l[^o].*|secrets\/[^/]+\/lo[^c].*|secrets\/[^/]+\/loc[^a].*|secrets\/[^/]+\/loca[^l].*|secrets\/[^/]+\/local[^/].*
+      kms: >-
+        arn:aws:kms:eu-central-1:689191102645:key/ca20bdac-ce35-4859-a2ef-e796df8d0abc
+      pgp: >-
+        15AC8EA84FC2A5AE768FFD753CEBBA453DE5BCFD,
+        33E4D64843E72C6C8A665703199BE5B70A2AACE3,
+        C5C51001A3BF0F4031C37EC99681FA65B010848C,
+        3A10EB97F2C401D93CC4685D6B0DDB72B8DEA45D
```

Comment:
Is this a private or public key?

Reply:
Those are public keys, the private ones are on each developers local machine. And one is on AWS.

Reply:
this is public